.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@AuraOfDivinity What is this for?

Reply:
@Rupeshiya It was for debugging purposes. My Bad I shouldn't have committed it.